cluster/ci/monitoring/mixins/prometheus/ghproxy.libsonnet

```diff
@@ -0,0 +1,45 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'ghproxy',
+        rules: [
+          {
+            alert: 'ghproxy-status-code-abnormal-%s' % std.strReplace(code, '..', 'XX'),
+            // excluding 404 because otherwise
+            // "this is going to spam us just about 24/7 with the OWNERS thing"
```

Comment:
Let's add a note in https://jira.coreos.com/browse/DPTP-447 to undo that

Reply:
Added